test/morrismethod.jl

```diff
@@ -6,6 +6,7 @@ f = @ode_def LotkaVolterra begin
 end a b c
   
 p = [1.5,1.0,3.0]
+t = collect(linspace(0,10,200))
 prob = ODEProblem(f,[1.0;1.0],(0.0,10.0),p)
-m = sample_matrices(prob,[[0,10],[0,10],[0,10]],[10,10,10],simulations=100)
+m = MorrisGlobalSensitivity(prob,Tsit5(),t,[[1,2],[1,2],[2,4]],[10,10,10],simulations=100)
 println(m)
```

Comment:
This is the test?

Reply:
do we have some indication that it's right?